src/app/qgstipfactory.cpp

```diff
@@ -258,8 +258,8 @@ QgsTipFactory::QgsTipFactory() : QObject()
   // by Harrissou Sant-anna
   myTip.setTitle( tr( "Set your own shortcuts for your actions" ) );
   myTip.setContent( tr( "QGIS provides you with a list of predefined shortcuts you can use to speed"
-                        " your workflow. These are available under <strong>Settings -> Configure"
-                        " shortcuts</strong> menu and can be extended and customized for any dialog or tool."
+                        " your workflow. These are available under <strong>Settings -> Keyboard"
+                        " Shortcuts</strong> menu and can be extended and customized for any dialog or tool."
```

Comment:
I'd suggest here:



     myTip.setContent( tr( "QGIS provides you with a list of predefined shortcuts you can use to speed"
                       " your workflow. These are available under <strong>Settings ->%1"
                    "</strong> menu and can be extended and customized for any dialog or tool.").arg( tr("Keyboard Shortcuts"));

That should keep the description synced with the translated menu name.